streamalert_cli/athena/helpers.py

```diff
@@ -89,11 +97,13 @@ def add_partition_statements(partitions, bucket, table_name):
     yield ' '.join(statement)
 
 
-def logs_schema_to_athena_schema(log_schema):
+def logs_schema_to_athena_schema(log_schema, ddl_statement=True):
```

Comment:
This is a minor nitpick:

Whenever I have optional parameters that control the "mode" of a function, I prefer to do something like this:


```
def logs_schema_to_athena_schema(log_schema):
  return _internal_function(log_schema, False)

def logs_schema_to_athena_ddl(log_schema):
  return _internal_function(log_schema, True)

# This method is private AND the "optional" statement is not optional, so there is no "default behavior"
def _internal_function(args, ddl_statement):  
  # ....
```

This allows you to avoid using default args, which can be really annoying

Reply:
It also allows you to search for the named function usages more easily in your codebase

Reply:
Personally, I don't want to add two more methods which can be simplified by adding a default argument.